VisualStudioPlugin/SolutionExplorerMenuCommand.cs

```diff
@@ -53,6 +53,11 @@ internal sealed class SolutionExplorerMenuCommand
 
         private ProjectFinder _projectFinder;
 
+        /// <summary>
+        /// Observer for the status of backtests launched from the plugin
+        /// </summary>
+        private IBacktestObserver _backtestObserver;
```

Comment:
should this field be marked as read-only?

Reply:
yes, good catch!